src/commands/infra/version.ts

```diff
@@ -0,0 +1,35 @@
+import commander from "commander";
+import fs from "fs";
+import shell from "shelljs";
+import { logger } from "../../logger";
+let bedrockDir: string = `${__dirname}/../../.bedrock/bedrock`;
+/**
+ * Adds the version command to the commander command object
+ *
+ * @param command Commander command object to decorate
+ */
+export const versionCommandDecorator = (command: commander.Command): void => {
+  command
+    .command("version")
+    .alias("v")
+    .description(
+      "Initialize will verify that all infrastructure deployment prerequisites have been correctly installed."
+    )
+    .action(async opts => {
+      const { silent = false, output } = opts;
+      shell.exec(
```

Comment:
Should we maybe move away from using shelljs? 

Since this is just performing a `terraform version`, I don't see this being too big of a problem.

Reply:
Deleting command since it was used for initial testing